tox.ini

```diff
@@ -103,6 +104,21 @@ commands =
     dbt run-operation upload_dbt_artifacts --args '\{filenames: [manifest, run_results], prefix: artifacts/schema_v4/snapshot/\}'
     dbt run-operation upload_dbt_artifacts --args '\{filenames: [manifest, sources], prefix: artifacts/schema_v4/source_freshness/\}'
     dbt run-operation upload_dbt_artifacts --args '\{filenames: [manifest, run_results], prefix: artifacts/schema_v4/test/\}'
+    # Retest the create macro without the replace option to make sure it's safe [and works]
+    dbt run-operation create_artifact_resources
+
+    dbt build -s dbt_artifacts --full-refresh
+    dbt build -s dbt_artifacts
+
+[testenv:integration_v2]
+changedir = integration_test_project
+commands =
+    dbt deps
+    # Setting the replace arg to tru so that we get a clean set of tabels on each run.
+    dbt run-operation create_artifact_resources --args '\{replace: true\}'
+    dbt run-operation upload_dbt_artifacts_v2
```

Comment:
Can we test uploading all of the files in the `schema_v4` directory here like the above integration tests do?

Reply:
👍